app/src/main/java/com/example/myapplication/MainActivity.java

```diff
@@ -84,6 +84,12 @@ public boolean onSupportNavigateUp() {
         return NavigationUI.navigateUp(navController, appBarConfiguration)
                 || super.onSupportNavigateUp();
     }
+    @Override
+    public void onBackPressed() { //This is to ensure that nothing happens when you press the back arrow.
+        if (false){ //Don't question this.
```

Comment:
its definitely questionable

Reply:
How can I make it less? Exactly, there is no other way.